src/worker.mjs

```
@@ -18,17 +18,26 @@ export default {
         }
       };
       const { pathname } = new URL(request.url);
+
+      console.log(`\n=== [Incoming Request] ===`);
+      console.log(`${request.method} ${request.url}`);
+
       switch (true) {
         case pathname.endsWith("/chat/completions"):
           assert(request.method === "POST");
-          return handleCompletions(await request.json(), apiKey)
+          const chatBody = await request.json();
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)